src/display/ui.rs

```diff
@@ -76,27 +79,57 @@ where
     }
     pub fn draw(&mut self, paused: bool) {
         let state = &self.state;
-        let ip_to_host = &self.ip_to_host;
+        let children = self.get_tables_to_display();
         self.terminal
             .draw(|mut frame| {
                 let size = frame.size();
-                let connections = Table::create_connections_table(&state, &ip_to_host);
-                let processes = Table::create_processes_table(&state);
-                let remote_addresses = Table::create_remote_addresses_table(&state, &ip_to_host);
                 let total_bandwidth = TotalBandwidth {
                     state: &state,
                     paused,
                 };
                 let help_text = HelpText { paused };
                 let layout = Layout {
                     header: total_bandwidth,
-                    children: vec![processes, connections, remote_addresses],
+                    children,
                     footer: help_text,
                 };
                 layout.render(&mut frame, size);
             })
             .unwrap();
     }
+    fn build_processes_table(&self) -> Table<'static> {
+        Table::create_processes_table(&self.state)
+    }
+
+    fn build_addresses_table(&self) -> Table<'static> {
+        Table::create_remote_addresses_table(&self.state, &self.ip_to_host)
+    }
+
+    fn build_connections_table(&self) -> Table<'static> {
+        Table::create_connections_table(&self.state, &self.ip_to_host)
+    }
+
+    fn get_tables_to_display(&self) -> Vec<Table<'static>> {
+        let opts = &self.opts;
+        let mut children: Vec<Table> = Vec::new();
+        if opts.processes {
+            children.push(self.build_processes_table());
```

Comment:
Can we use `Table::create_processes_table(...)` here directly? Then we can get rid of the `build_*_table` methods above.